mobile/android/LogcatHelper.ts

```diff
@@ -0,0 +1,59 @@
+///<reference path="../../../.d.ts"/>
+"use strict";
+import byline = require("byline");
+
+function getConsoleLogFromLine(lineText:String):any {
+	var acceptedTags = ["chromium", "Web Console", "JS"];
+
+	//sample line is "I/Web Console(    4438): Received Event: deviceready at file:///storage/emulated/0/Icenium/com.telerik.TestApp/js/index.js:48"
+	var match = lineText.match(/.\/(.+?)\s*\(\s*(\d+?)\): (.*)/);
+	if(match) {
+		if(acceptedTags.indexOf(match[1]) !== -1) {
+			return {tag: match[1], message: match[3]};
+		}
+	}
+	else if(_.any(acceptedTags, (tag:string) => {
+			return lineText.indexOf(tag) !== -1;
+		})) {
+		return {message: match[3]};
+	}
+
+	return null;
+}
+
+class LogcatHelper {
```

Comment:
Consider creating an interface ILogcatHelper and implement it with this class. resolutions through the injector should deal with this interface only.